miniapps/tools/lor-transfer.cpp

```diff
@@ -95,6 +96,9 @@ int main(int argc, char *argv[])
    args.AddOption(&use_pointwise_transfer, "-t", "--use-pointwise-transfer",
                   "-no-t", "--dont-use-pointwise-transfer",
                   "Use pointwise transfer operators instead of L2 projection.");
+   args.AddOption(&forceL2transfer, "-fl2x", "--force-l2-transfer",
+                  "-no-fl2x", "--dont-force-l2-transfer",
+                  "Force L2 projection assuming L2 finite element space (even if H1).");
```

Comment:
I'm not sure this option is necessary, I think it can be removed from the miniapp.

Reply:
Agreed.  Fixed